### PR TITLE
Holiday season

### PR DESCRIPTION
**Pull request by gmauro**

### CODE REVIEW

File: _data/notices.yml [hunk 1]
```diff
@@ -50,6 +50,23 @@
             
             If you need help submitting your data to public archives, like ENA, please [get in touch](mailto:contact@usegalaxy.eu). We will support you in sharing your data.
 
+- title: Holiday season
+  class: info # one of [success, info, warning, danger]
+  site: freiburg # Your site
+  messages:
+      - message: |
+            Galaxy Europe will continue to operate over the holiday season however, with reduced support capacity. This period will run from 24th December 2021 until the 4th of January 2022.
```
Comment (beatrizserrano) [inline _data/notices.yml:58]:
```suggestion
            Galaxy Europe will continue to operate over the holiday season although with reduced support capacity. This period will run from 24th December 2021 until the 4th of January 2022.
```

File: _data/notices.yml [hunk 2]
```diff
@@ -50,6 +50,23 @@
...
+            During this time:
+                No new tools or reference data will be installed
+                No requests for disk space will be actioned
+                No support tickets will be responded to
+                No TIaaS requests will be evaluated
+
+            We will also have limited service administration capacity and while we will make all reasonable efforts to keep the service running there may be issues outside of our control during this period.
```
Comment (beatrizserrano) [inline _data/notices.yml:65]:
```suggestion
            We will also have limited service administration capacity and while we will make all reasonable efforts to keep the service running, there may be issues outside of our control during this period.
```

File: _data/notices.yml [hunk 3]
```diff
@@ -50,6 +50,23 @@
...
 
+- title: Holiday season
+  class: info # one of [success, info, warning, danger]
+  site: freiburg # Your site
+  messages:
+      - message: |
+            Galaxy Europe will continue to operate over the holiday season however, with reduced support capacity. This period will run from 24th December 2021 until the 4th of January 2022.
+            During this time:
+                No new tools or reference data will be installed
+                No requests for disk space will be actioned
```
Comment (anuprulez) [inline _data/notices.yml:61]: No requests for disk space will be completed

Comment (anuprulez) [inline _data/notices.yml:61]:
```suggestion
                No requests for disk space will be completed
```

**Review by beatrizserrano — APPROVED** [review]

**Pull request merged**